src/base/componentwise.rs

```diff
@@ -43,7 +44,45 @@ impl<T: Scalar, R: Dim, C: Dim, S: Storage<T, R, C>> Matrix<T, R, C, S> {
         res
     }
 
-    // TODO: add other operators like component_ln, component_pow, etc. ?
+    /// Computes the component-wise minimum
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use nalgebra::Vector3;
+    ///
+    /// let u = Vector3::new(-1.0, 3.0, 2.0);
+    /// let v = Vector3::new(1.0, 2.0, 3.0);
+    /// assert_eq!(u.component_min(&v), Vector3::new(-1.0, 2.0, 2.0))
+    /// ```
+    pub fn component_min(&self, rhs: &Matrix<T, R, C, S>) -> OMatrix<T, R, C>
+    where
+        T: PartialOrd + Copy,
```

Comment:
`T` doesn't have to be `Copy`, `Clone` is enough and will be optimized away by the compiler.